crypto/ocsp/ocsp_vfy.c

```diff
@@ -85,6 +86,9 @@ int OCSP_basic_verify(OCSP_BASICRESP *bs, STACK_OF(X509) *certs,
             OCSPerr(OCSP_F_OCSP_BASIC_VERIFY, ERR_R_X509_LIB);
             goto f_err;
         }
+        if ((flags & OCSP_PARTIAL_CHAIN) != 0
+                && (vp = X509_STORE_CTX_get0_param(ctx)) != NULL)
+            X509_VERIFY_PARAM_set_flags(vp, f | X509_V_FLAG_PARTIAL_CHAIN);
```

Comment:
Certainly good to make use of the 'or' semantics of ` X509_VERIFY_PARAM_set_flags()`.
Yet I'm puzzled where the `f |` part comes wrong, likely a leftover which should not even compile.

Reply:
gaa, didn't save the buffer before doing the commit.  updated commit pushed.